tests/mps_statespace_test.cc

```diff
@@ -900,6 +900,218 @@ TEST(MPSStateSpaceTest, ReduceDensityMatrixLarge){
 
 }
 
+TEST(MPSStateSpaceTest, SampleOnceSimple){
```

Comment:
This could be parameterized with [TEST_P](https://google.github.io/googletest/reference/testing.html#TEST_P).

Reply:
It could, but I see it as being just on the cusp of worth the extra test boilerplate for switching to TEST_P :stuck_out_tongue: . Do you feel strongly we should make the switch ?

Reply:
It's been a while since I've had to do this in C++, so I don't recall all the boilerplate needed. If you don't see it as worthwhile I'm fine keeping it as-is.